include/range/v3/utility/iterator.hpp

```diff
@@ -189,7 +189,7 @@ namespace ranges
             {
                 RANGES_EXPECT((Same<I, S>() || 0 <= n));
                 D d = bound - it;
-                RANGES_EXPECT(0 <= n ? 0 <= d : 0 >= d);
+                //RANGES_EXPECT(0 <= n ? 0 <= d : 0 >= d);
```

Comment:
oops, this shouldn't haven't been commented out. I had a lot of debugging logic in there when i was trying to reverse engineer what `distance_to(view const& other)` actually was supposed to do when doing the RandomAccess version.